src/stavetempo.ts

```diff
@@ -4,13 +4,38 @@
 import { Flow } from './tables';
 import { StaveModifier } from './stavemodifier';
 import { Glyph } from './glyph';
+import { FontInfo } from './types/common';
+import { Stave } from './stave';
+
+export interface StaveTempoOptions {
+  bpm: number;
+  dots: number;
+  duration: string;
+  name: string;
+}
+
+export interface Tempo {
```

Comment:
Both `StaveTempoOptions` and `Tempo` are the same -- let's just use `StaveTempoOptions`.

Reply:
done